common/snapshots.py

```diff
@@ -3087,6 +3097,25 @@ def lastSnapshot(cfg):
         return sids[0]
 
 
+def has_missing(included):
+    """
+    Check if there are missing files or folders in a snapshot.
+
+    Args:
+        included (list):    list of tuples (item, info)
+
+    Returns:
+        tuple:              (bool, str) where bool is ``True`` if there are
+                            missing files or folders and str is a message
+                            describing the missing files or folders
+    """
+    not_found = []
+    for path, info in included:
+        if not os.path.exists(path):
```

Comment:
I am not sure if `os.path.exists()` does also work for files (though folders are files), could you please (manually) check this with a configuration that contains a "missing" and an "existing" file (positive and negative test)?

Reply:
Sure. I just did it and it worked fine :)

I tested via CLI:

```
➜  /tmp cd /tmp                                                                                                                                                                                                                          /0.0s
➜  /tmp touch test.txt                                                                                                                                                                                                                   /0.0s
➜  /tmp python
Python 3.12.4 (main, Jun  7 2024, 06:33:07) [GCC 14.1.1 20240522] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import os
>>> os.path.exists("/tmp")
True
>>> os.path.exists("/tmp/test.txt")
True
>>> os.path.exists("/wrong")
False
>>> os.path.exists("/tmp/wrong.txt")
False
```

I tested w/ CLI, and got correct results w/ this warning:

```
WARNING: The following **files/**folders are missing: /home/rafaelhdr/RMFolder, /home/rafaelhdr/RMFIle
```

And also via GUI with real snapshots (w/ existent folder, removed folder, existent file and removed file). 

![Screenshot from 2024-07-26 10-06-02](https://github.com/user-attachments/assets/b46207f0-f8fb-4f9a-b645-471e02934e91)